src/datomish/query.cljc

```diff
@@ -127,10 +133,10 @@
 (defn apply-elements-to-context [context elements]
   (assoc context :elements elements))
 
-(defn patterns->context
-  "Turn a sequence of patterns into a Context."
-  [patterns]
-  (reduce apply-pattern-to-context (make-context) patterns))
+(defn patterns->into-context
```

Comment:
These should be `apply-patterns-to-context`, I think. Certainly not both `->` ("transform") and `into`.